### PR TITLE
Add configurable ICMP identifier

### PR DESCRIPTION
**Pull request by natesales**
This commit adds two functions to set and get a `Pinger`'s ICMP identifier field.

### CODE REVIEW

**Review by SuperQ — COMMENTED** [review]
Good idea.

I wonder if we should also expose the `currentUUID` as well.

File: ping.go [hunk 1]
```diff
@@ -232,6 +232,9 @@ type Packet struct {
 
 	// TTL is the Time To Live on the packet.
 	Ttl int
+
+	// Id is the ICMP identifier.
+	Id int
```
Comment (SuperQ) [inline ping.go:237]:
```suggestion
	ID int
```

File: ping.go [hunk 2]
```diff
@@ -385,6 +388,16 @@ func (p *Pinger) SetLogger(logger Logger) {
 	p.logger = logger
 }
 
+// SetID sets the ICMP identifier
```
Comment (SuperQ) [inline ping.go:391]:
```suggestion
// SetID sets the ICMP identifier.
```

File: ping.go [hunk 3]
```diff
@@ -385,6 +388,16 @@ func (p *Pinger) SetLogger(logger Logger) {
...
+func (p *Pinger) SetID(id int) {
+	p.id = id
+}
+
+// ID returns the ICMP identifier
```
Comment (SuperQ) [inline ping.go:396]:
```suggestion
// ID returns the ICMP identifier.
```

**Comment by natesales** [comment]
Thanks for the review! I've made the proposed changes and squashed into one commit.

**Review by SuperQ — APPROVED** [review]
LGTM

**Review by CHTJonas — APPROVED** [review]
LGTM

**Pull request merged**